### PR TITLE
symbolic numeric blended command

### PR DESCRIPTION
**Pull request by SamDuffield**
More robust symbolic implementation (for QAOA style gates that have both symbolic and numeric parameters)

Modified README

### CODE REVIEW

File: .github/workflows/docs/build-docs [hunk 1]
```diff
@@ -96,4 +111,4 @@ if __name__ == "__main__":
             MODULES_DIR / "docs" / "build",
             dest,
             dirs_exist_ok=True,
-            )
```
Comment (cqc-melf) [inline .github/workflows/docs/build-docs:99]: Why are you changing this line?

File: pytket/extensions/qujax/qujax_convert.py [hunk 2]
```diff
@@ -36,9 +39,79 @@ def _tk_qubits_to_inds(tk_qubits: Sequence[Qubit]) -> Tuple[int, ...]:
     return tuple(q.index[0] for q in tk_qubits)
 
 
+def _append_func(f: Callable, func_to_append: Callable) -> Callable:
```
Comment (cqc-melf) [inline pytket/extensions/qujax/qujax_convert.py:42]: Functions with names starting with `_` will not be shown in the documentation, is that intended?

Comment (SamDuffield) [reply pytket/extensions/qujax/qujax_convert.py:42]: yep!

File: .github/workflows/docs/build-docs (hunk 1 above)
Comment (SamDuffield) [reply .github/workflows/docs/build-docs:99]: that one was PyCharm 👀
I can change it back if you like?

File: docs/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,13 @@
 Changelog
 ~~~~~~~~~
 
+0.6.1 (September 2022)
+-------------------
```
Comment (cqc-melf) [inline docs/changelog.rst:5]:
```suggestion
----------------------
```

File: _metadata.py [hunk 4]
```diff
@@ -1,2 +1,2 @@
-__extension_version__ = "0.6.0"
+__extension_version__ = "0.6.1"
```
Comment (cqc-melf) [inline _metadata.py:1]: With an update of the qujax version I would change to 0.7.0
```suggestion
__extension_version__ = "0.7.0"
```

File: tests/test_tket.py [hunk 5]
```diff
@@ -54,7 +54,7 @@ def _test_circuit(
         circuit_commands = [
             com for com in circuit.get_commands() if str(com.op) != "Barrier"
         ]
-        circuit_2 = qujax_args_to_tk(*tk_to_qujax_args(circuit))
+        circuit_2 = qujax_args_to_tk(*tk_to_qujax_args(circuit))  # type: ignore
```
Comment (cqc-melf) [inline tests/test_tket.py:57]: Why is this type ignore here required? Have you tried a cast? The return type of the function should be `circuit`?

File: tests/test_tket_symbolic.py [hunk 6]
```diff
@@ -138,12 +138,28 @@ def test_symbolic_numeric_blend() -> None:
...
+
+
 def test_not_in_qujaxgates() -> None:
     symbols = [Symbol("p0")]  # type: ignore
 
-    circuit = Circuit(2)
+    circuit = Circuit(3)
     circuit.Rx(symbols[0], 0)
-    circuit.ZZPhase(0.1, 0, 1)
+    circuit.XXPhase3(0.1, 0, 1, 2)
```
Comment (cqc-melf) [inline tests/test_tket_symbolic.py:162]: Is there a reason why this is changed?

Comment (SamDuffield) [reply tests/test_tket_symbolic.py:162]: Yes ZZPhase has been added to qujax in 0.2.8 and this test is supposed to test a gate not found in qujax

File: _metadata.py (hunk 4 above)
Comment (SamDuffield) [reply _metadata.py:1]: ok!

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**